pandas/tests/plotting/test_graphics_others.py

```diff
@@ -18,14 +18,15 @@
 
 import pandas.tools.plotting as plotting
 
-from pandas.tests.test_graphics import (TestPlotBase, _check_plot_works,
-                                        curpath, _ok_for_gaussian_kde)
+from pandas.tests.plotting.common import (TestPlotBase, _check_plot_works,
+                                          curpath, _ok_for_gaussian_kde)
 
 
 """
 These tests are for ``DataFrame.hist``, ``DataFrame.boxplot`` and
 other miscellaneous plots.
-`Dataframe.plot`` and ``Series.plot`` are tested in test_graphics.py
+`Dataframe.plot`` and ``Series.plot`` are tested in
+test_frame.py and test_series.py
```

Comment:
@TomAugspurger I don't think that is a problem, indeed easy to make a base class,  and there is already a common file

@sinhrks Is there a reason to call it `test_legacy_hist` instead of `test_hist`? As we did not really officially deprecate them? And they are still used in the docs (but that is a very minor comment :-))